keel/leads/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+from keel.authentication.models import TimeStampedModel
+
+
+class CustomerLead(TimeStampedModel):
+
+    LEAD_SOURCE = (
+        (1, 'WEB'),
+        (2, 'SALES'),
+        (3, 'SOCIAL')
+    )
+
+    RESOLUTION = (
+        (1, 'CONVERTED'),
+        (2, 'DROPPED')
+    )
+
+    lead_source = models.PositiveSmallIntegerField(verbose_name="Lead Source", choices=LEAD_SOURCE)
+    resolution = models.PositiveSmallIntegerField(verbose_name="Resolution", choices=RESOLUTION)
+    email = models.EmailField(verbose_name="Email", unique=True, blank=False, null=True, default=None)
+    phone_number = models.BigIntegerField(verbose_name='Phone Number', default=None, blank=False, null=True)
+    is_active = models.BooleanField(verbose_name="Is Active", default=True)
+
+
+    def __str__(self):
+        return self.phone_number
```

Comment:
we are good to go, just add a Meta class to the model where we can define the db_table attribute
because this way we can name the tables as per our convenience, else django names them in a legacy way
follow this for more help
https://docs.djangoproject.com/en/3.2/topics/db/models/

Reply:
Alright, I've done that

Reply:
Do I need to re open another PR?

Reply:
ok...did it create any new migration?

Reply:
No, I didn't. Should I?

Reply:
yes, you shouldnot push any code with any left over migrations, Please check if this meta class has created any new migration...and the push accordingly and update

Reply:
Okay, ,Thanks. I have added the migrations